docs/index.rst

```diff
@@ -49,32 +49,53 @@ Questions can also be asked on
 Some recent development changelog can be seen `here <https://github.com/rwth-i6/returnn/blob/master/CHANGELOG.md>`__.
 
 
-User guide
-----------
+.. Getting Started
+.. ---------------
 
 .. toctree::
+    :hidden:
+    :caption: Getting Started
+    :maxdepth: 2
+
+    getting_started/installation
+    getting_started/basic_usage/basic_usage
+    getting_started/framework.rst
+    getting_started/faq.rst
+
+.. toctree::
+    :hidden:
+    :caption: Reference
     :maxdepth: 2
 
-    installation
-    basic_usage
-    tech_overview
-    tf_lstm_benchmark
-    multi_gpu
-    deterministic_training
-    debugging
-    profiling
     configuration_reference/index.rst
     dataset_reference/index.rst
     layer_reference/index.rst
 
+.. toctree::
+    :hidden:
+    :caption: Advanced Topics
+    :maxdepth: 2
+
+    advanced/pretraining.rst
+    advanced/multi_gpu.rst
+    advanced/debugging.rst
+    advanced/profiling.rst
+    advanced/deterministic_training.rst
 
-API Reference
--------------
+.. toctree::
+    :hidden:
+    :caption: Applications
+
+    applications/asr.rst
+    applications/lm.rst
+    applications/mt.rst
 
 .. toctree::
-    :maxdepth: 2
-    :glob:
+    :hidden:
+    :caption: Internals
 
+    internals/tech_overview.rst
+    internals/tf_lstm_benchmark.rst
```

Comment:
I don't know if fits under "internal".

Reply:
I did not find a good place for it yet, maybe adding a "features" subsection under "getting started" that has a list of what RETURNN can do (or not do) would be an appropriate place.

Reply:
I would move the technological overview to the `getting_started` directory.

And I would move the LSTM benchmark also somewhere else. Maybe `details`? Or `extended_usage`? This is really supposed to be for users, to give them an idea of what LSTM cell/unit they should use.

Reply:
Okay I put this to the beginning now, the network part is still redundant, but I will keep it for now until further modifications of the overview. For the LSTM benchmark I was still not sure, so I just put it at the end of getting started.